common/source/host/mmd_device.cpp

```diff
@@ -18,11 +18,10 @@
 #include "afu_bbb_util.h"
 
 #define MMD_COPY_BUFFER_SIZE (2 * 1024 * 1024)
-#define MSGDMA_BBB_GUID   "BC24AD4F-8738-F840-575F-BAB5B61A8DAE"
-#define MSGDMA_BBB_SIZE 256
+#define DMA_BBB_GUID   "BC24AD4F-8738-F840-575F-BAB5B61A8DAE"
```

Comment:
Please add comments explaining use of different IDs. Have all ID in single/related (e.g. dma.h & mmd_device.h) header file instead of separate files.